log/log.go

```diff
@@ -14,34 +16,98 @@ import (
 var (
 	log      *zap.SugaredLogger
 	errorLog *os.File
+	output   string
+	logLevel sync.Map // this is a thread-safe map[string]string
+)
+
+const (
+	defaultLevel  = "*:error"
+	defaultOutput = "stderr"
 )
 
 func init() {
 	// Allow overriding the default log level via $LOG_LEVEL, so that the
 	// environment variable can be set globally even when running tests.
 	// Always initializing the logger is also useful to avoid panics when
 	// logging if the logger is nil.
-	level := "error"
+	level := []string{defaultLevel}
 	if s := os.Getenv("LOG_LEVEL"); s != "" {
-		level = s
+		level = strings.Split(s, ",")
 	}
-	Init(level, "stderr")
+
+	InitWithLevels(level, defaultOutput)
+}
+
+// Init initializes the logger, with a single logLevel (debug|info|warn|error|fatal).
+// out can be either "stdout", "stderr" or "/path/to/file" and it's a global setting.
+//
+// kept for backwards compatibility, it just wraps InitWithLevels()
```

Comment:
just modify Init to take a list? I don't think we need to worry about backwards compatibility in the log package, if the changes are pretty easy to adapt to.

Reply:
good point, done